src/libraries/System.Security.Cryptography.Algorithms/tests/Rfc2898OneShotTests.cs

```diff
@@ -11,6 +11,7 @@ namespace System.Security.Cryptography.DeriveBytesTests
     [SkipOnMono("Not supported on Browser", TestPlatforms.Browser)]
     public static class Rfc2898OneShotTests
     {
+        // [SuppressMessage("Microsoft.Security", "CS002:SecretInNextLine", Justification="Password for testing.")]
```

Comment:
in general when we suppress we should note that exemption was approved. That way we can spot when someone adds one without approval